loadtest/util/page.ts

```diff
@@ -1,23 +1,55 @@
-import { check, fail } from "k6";
+import { check } from "k6";
 import http from "k6/http";
+import login from "../usecases/1_login.ts";
+import goToStart from "../usecases/1_show-start.ts";
 import { getFrontendUrl } from "../util/config.ts";
 import { loadLinkedResourcesAndCheck } from "../util/load-linked-resources.ts";
+import {
+  getLoginInfo,
+  getOrganisationen,
+  getPersonenIds,
+  getPersonenUebersicht,
+  getRollen,
+  getServiceProviderLogos,
+  getServiceProviders,
+} from "./api.ts";
 import { defaultTimingCheck } from "./checks.ts";
+import { getDefaultAdminMix, UserMix } from "./users.ts";
 
 const frontendUrl = getFrontendUrl();
 
 export function goToStartPage() {
   loadPage(frontendUrl);
 }
 
-export function goToOIDCServiceProvider() {
-  // TODO: add url
-  const url = frontendUrl;
-  fail("url needs to be added");
-  loadPage(url);
+export function navigateToHomepage(users: UserMix) {
+  login(users);
+  goToStartPage();
+  getLoginInfo();
+  const providers = getServiceProviders();
+  getServiceProviderLogos(providers);
+  return providers;
 }
 
-function loadPage(url: string) {
+export function navigateToUserList(users = getDefaultAdminMix()) {
+  goToStart(users);
+  getLoginInfo();
+  getOrganisationen([
+    "limit=25",
+    "systemrechte=PERSONEN_VERWALTEN",
+    "excludeTyp=KLASSE",
+  ]);
+
+  // TODO: see if this behaviour should be emulated
+  for (let i = 0; i < 2; i++) {
+    const personIds = getPersonenIds();
+    getPersonenUebersicht(personIds);
+  }
```

Comment:
Is there a follow-up ticket for this TODO? Add the ticket ID here

Reply:
Clarified offline with architects. The behavior is undesirable, but will stay the same for the near future and thus I'll keep it in these tests too. Comment was removed.